app/views/toolbar.jade

```diff
@@ -41,15 +41,24 @@ nav.navbar.navbar-inverse.navbar-top
                         ul.dropdown-menu(aria-labelledby='dropdownGames')
                             li(ng-repeat='game in games')
                                 a(ng-if='isDeveloper()' ng-href='data?game={{game._id}}' ng-click='setSelectedGame(game)') {{game.title}}
-                                a(ng-if='!isDeveloper()' ng-href='class?game={{game._id}}' ng-click='setSelectedGame(game)') {{game.title}}
+                                a(ng-if='!isDeveloper()' ng-href='gameactivity?game={{game._id}}' ng-click='setSelectedGame(game)') {{game.title}}
                 li.dropdown
-                    li(ng-if='isUser()&&isTeacher()&&hasSessions()')
+                    li(ng-if='isUser()&&hasClasses()')
```

Comment:
Fixed in: https://github.com/e-ucm/rage-analytics-frontend/pull/90/commits/5353294c7dba6c4720c637a730bf89ca7c7cbfb0